application-ideas-ui/src/main/resources/Ideas/RecentIdeasMacro.xml

```diff
@@ -251,7 +251,7 @@
         #end
         [[$document.title&gt;&gt;$idea]] $services.localization.render('ideas.author', ["[[$xwiki.getUserName($document.author, false)&gt;&gt;$document.author]]"]) $services.localization.render('ideas.date', [$xwiki.formatDate($document.date, 'dd/MM/yyyy hh:mm')])
         &lt;span class="hidden status"&gt;$document.status&lt;/span&gt;
-        &lt;span class="hidden fullName"&gt;$document.fullName&lt;/span&gt;
+        &lt;span class="hidden fullName"&gt;$document.documentReference&lt;/span&gt;
```

Comment:
If the idea document reference is not passed for display purpose (which would be strange) then it's better to pass it using a ``data-*`` (e.g. ``data-reference=""``) attribute on the ``div.ideas`` above. Same for the status, if not used for display (e.g. ``data-status=""``).